packages/@aws-cdk/aws-codebuild/README.md

```diff
@@ -75,7 +75,24 @@ You can also add the Project to the Pipeline directly:
 
 ```ts
 // equivalent to the code above:
-project.addBuildToPipeline(buildStage, 'CodeBuild');
+const buildAction = project.addBuildToPipeline(buildStage, 'CodeBuild');
+```
+
+In addition to the build Action,
+there is also a test Action.
+It works very similarly to the build Action,
+the only difference is that the test Action never produces any output artifacts.
+
+Examples:
+
+```ts
+new codebuild.PipelineTestAction(this, 'CodeBuild', {
```

Comment:
Use a more meaningful ID in the example, something like `IntegrationTest`

Reply:
Done.